README.md

```diff
@@ -8,15 +8,29 @@ This repo features:
 We expect it to feature:
 * A Quarto presentation theme that conforms to SU branding
 
-We may end up splitting off some of these elements to make them easier to deploy, we haven't quite thought it through yet. If stuff does move around we'll update the README
+## Contributing to this repo
+
+* Request access from a member of the Data Science team, they need to add you as a contributor
```

Comment:
Could link to `about.qmd` so people can see who are the members of the Data Science team.